### ISSUE TITLE
Bedrock players get instantly kicked

### ISSUE THREAD

**Issue by Demirramon** [GeyserMC/Geyser#3746]
### Describe the bug

Bedrock players cause an uncaught exception as soon as they join a server and they disconnect after a few seconds.

This is the exception that shows up (usually multiple times in a row) before the disconnect:
```
[WARN] [io.netty.channel.DefaultChannelPipeline] An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
io.netty.channel.unix.Errors$NativeIoExceptionsendmmsg(..) failed: Operation not permitted
```

Server log: https://mclo.gs/ozOMJQi

### To Reproduce

1. Join server normally
2. World loads normally but the server stops responding immediately
3. Exceptions start showing in the console
4. After a few seconds, the player is disconnected

### Expected behaviour

The player should be able to play normally but is unable to communicate with the server and then gets disconnected.

### Server Version and Plugins

This server is running Paper version git-Paper-524 (MC: 1.19.4) (Implementing API version 1.19.4-R0.1-SNAPSHOT) (Git: 9846d0d)

The plugin list is included in the Geyser dump.

### Geyser Dump

https://dump.geysermc.org/seMRlbCTitPesZXMuguKzuM0hgKSaJ7N

### Geyser Version

2.1.0-SNAPSHOT (git-master-6e9b272)

### Minecraft: Bedrock Edition Device/Version

Windows 10 and Android

**Comment by Versimno** [comment]
I'm getting the same exact issue on my fabric server

```


[epollEventLoopGroup-3-2/WARN]: An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
--
467 | io.netty.channel.unix.Errors$NativeIoException: sendmmsg(..) failed: Operation not permitted


``` 

geyser dump:[https://dump.geysermc.org/7VMPEcQ5PlttqA6aLaaoucL0WQTGnxMw](https://dump.geysermc.org/7VMPEcQ5PlttqA6aLaaoucL0WQTGnxMw) 
latest log: [https://mclo.gs/MFtoByJ](https://mclo.gs/MFtoByJ) 

Minecraft 1.19.4 with Fabric Loader 0.14.19
Geyser version 2.1.0-SNAPSHOT (git-master-9b264e6)

**Comment by Novasky-Guy** [comment]
Getting the same issue with a Waterfall server
https://dump.geysermc.org/jJP2bMBMLEca2r7KhQnXU1ksaCKDI5xB

[11.05 22:12:46] [Server] io.netty.channel.unix.Errors$NativeIoExceptionsendmmsg(..) failed: Operation not permitted
[11.05 22:12:46] [Server] [io.netty.channel.DefaultChannelPipeline]An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.

Latest version of everything

**Comment by Demirramon** [comment]
I managed to solve the problem! I don't know if Geyser itself has anything to do with it, but it turns out something was wrong with the network of the server host.

Leaving this here for people using hosts: I use BisecHosting and all I needed to do is to turn off the UDP Network (required to use multiple ports) and then turn it on again. After that, it worked first try. Other hosts may use different names, but it may be the same thing.

I'm not sure if I should close this issue or not so I will leave it for the devs to decide.
